tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -63,19 +65,18 @@ spec:
       # end
   - name: run-loadtest
     image: 197575167141.dkr.ecr.us-west-2.amazonaws.com/clusterloader2:76e3fd7
-    command: ["./clusterloader"]
-    args: 
-    - --kubeconfig=$(workspaces.config.path)/kubeconfig
-    - --testconfig=$(workspaces.source.path)/perf-tests/clusterloader2/testing/load/config.yaml
-    - --testoverrides=$(workspaces.source.path)/overrides.yaml
-    - --nodes=$(params.nodes) 
-    - --provider=eks 
-    - --report-dir=$(workspaces.results.path)
-    - --alsologtostderr
-    env:
-    - name: ENABLE_EXEC_SERVICE
-      value: "false"
+    script: |
+      ENABLE_EXEC_SERVICE=false ./clusterloader --kubeconfig=$(workspaces.config.path)/kubeconfig --testconfig=$(workspaces.source.path)/perf-tests/clusterloader2/testing/load/config.yaml --testoverrides=$(workspaces.source.path)/overrides.yaml --nodes=$(params.nodes) --provider=eks --report-dir=$(workspaces.results.path) --alsologtostderr
+      #To run the next step regardless of success/failure of the test
+      TEST_EXIT_CODE=$?
+      if [ $TEST_EXIT_CODE != 0 ]; then
+        exit 0
+      fi
+    timeout: 30000s
```

Comment:
This also is a very long time to run these tests

Reply:
Trust me, it ran out even on this timeout because of instability of KIT controlplane when testing megaxl load :D

Reply:
A successful test with 10 load throughput would take around 6-7hrs. We need logs and data from the failed tests which could run up to this time. If i see this is causing issues, i'll reduce it in future, I think this is good for now.